infrastructure/dpladm/env-repo-template/standard/.lagoon.yml

```diff
@@ -56,7 +56,13 @@ tasks:
         name: Import config translations
         command: drush dpl_po:import-remote-config-po da https://danskernesdigitalebibliotek.github.io/dpl-cms/translations/da.config.po
         service: cli
-
+    - run:
+        name: Set Pod resource requests and limits
+        command: |
+            set -e;
+            echo "setting resource requests for cli"
+            kubernetes patch deployment cli -n ${LAGOON_PROJECT_NAME} -p '{"spec":{"template":{"spec":{"containers":[{"name":"cli","resources":{"requests":{"memory":"20Mi"}}}]}}}'
```

Comment:
I wonder what this resource request is about.

- Why is is handled separately from `infrastructure/task/scripts/adjust-resource-requests.sh`?
- What is the origin of the `20Mi` value?

Reply:
Good catch - This should have been removed as it doesn't work